### DISCUSSION TITLE
Do you hav zh-CN language in the Web?

### DISCUSSION THREAD

**Discussion by 164ebr** [plan-player-analytics/Plan#2395] [Archive]
### Question

None

### Additional context

None

**Comment by AuroraLS3** [comment]
Here is the list of available languages https://github.com/plan-player-analytics/Plan/tree/master/Plan/common/src/main/resources/assets/plan/locale

There is ZH_TW and CN, but not ZH_CN - Feel free to contribute if that is necessary https://github.com/plan-player-analytics/Plan/wiki/Localization
